code/API_definitions/location-retrieval.yaml

```diff
@@ -348,55 +348,83 @@ components:
         message:
           type: string
           description: Detailed error description
+
   responses:
-    RetrieveLocationBadRequest400:
-      description: |-
-        Problem with the client request. In addition to regular scenario of `INVALID_ARGUMENT`, another scenarios may exist:
-          - maxAge threshold cannot be satisfied
+    Generic400:
+      description: Bad Request
       headers:
         x-correlator:
-          $ref: '#/components/headers/x-correlator'
+          $ref: "#/components/headers/x-correlator"
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/ErrorInfo'
+            $ref: "#/components/schemas/ErrorInfo"
           examples:
-            InvalidArgument:
+            GENERIC_400_INVALID_ARGUMENT:
+              summary: Generic Invalid Argument
+              description: Invalid Argument. Generic Syntax Exception
               value:
                 status: 400
                 code: INVALID_ARGUMENT
-                message: "Invalid argument"
-            MaxAgeIssue:
+                message: Client specified an invalid argument, request body or query param.

```

Comment:
In theory this also applies to request headers. Maybe we can simplify the sentence, keep it more generic ?

Reply:
Hello @alpaycetin74 
I used the the template from commonalities [here](https://github.com/camaraproject/Commonalities/blob/main/artifacts/CAMARA_common.yaml) - line 146.... so probably we cannot change this.

Reply:
Also, the value for message in examples are not normative, only the code. Implementation may return any other, even in different language